crates/turborepo-ui/src/lib.rs

```diff
@@ -50,6 +50,10 @@ impl UI {
         Self { should_strip_ansi }
     }
 
+    pub fn is_ci() -> bool {
+        turborepo_ci::is_ci() && std::io::stdout().is_terminal()
+    }
```

Comment:
This function is misplaced. Looking at Go since this is only used by `run.go`, I'd much rather remove `UI::is_ci` and change the usage in `run` to explicitly check if stdout is a terminal.

Reply:
Also is there any reason this isn't included in the general CI check?

Reply:
Not to my knowledge, not sure why we'd want to behave differently on CI depending on if we detect tty.